packages/platforms/browser/lib/browser.ts

```diff
@@ -51,7 +51,8 @@ const BugsnagPerformance = createClient({
       performance
     ),
     new NetworkRequestPlugin(spanFactory, fetchRequestTracker, xhrRequestTracker),
-    new RouteChangePlugin(spanFactory, window.location)
+    new RouteChangePlugin(spanFactory, window.location),
+    new ResourceLoadPlugin(spanFactory, window.PerformanceObserver)
```

Comment:
Worth a comment that the order is important if we're relying on this being after the page load plugin for parentage